notification/templates/notification/notification_list.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'layout.html' %}
+{% load static %}
+
+{% block head_end %}
+    {{ block.super }}
+    <link href="{% static 'notification/css/notification.css' %}" rel="stylesheet"/>
+{% endblock %}
+
+{% block content %}
+    {% csrf_token %}
+    <div class="row mt-4">
+        <div class="jumbotron col">
+            <h1 class="display-4">Notifications</h1>
+            <hr />
+            <p>If you would like to change the way DAISY handles your notifications, please update your <a href="{% url 'notifications_settings' %}">notifications settings</a>.</p>
+            {% if not show_dismissed %}
+                <p>By default, dismissed notifications are not shown. You can display dismissed notifications by clicking <a href="{{ request.path }}?show_dismissed=true">here</a></p>
+            {% endif %}
```

Comment:
The links can be surround by <strong> tags or have a font weight bold similar to user guide link in homepage
<img width="412" alt="Screenshot 2023-11-08 at 14 11 26" src="https://github.com/elixir-luxembourg/daisy/assets/33813190/7eac26cc-886a-4c3a-b0ea-d4e1cb032cf6">

Reply:
Corrected

Reply:
thank you!